lib/builder.js

```diff
@@ -240,7 +240,10 @@ function addExtraOutputs(output, tree, opts) {
   });
 }
 
-Builder.prototype.buildTree = function(tree, outFile, opts) {
+/**
+ * @param {Object} [cache] - Optional object used to cache module outputs.
+ */
+Builder.prototype.buildTree = function(tree, outFile, opts, cache) {
```

Comment:
Yeah I considered putting it there (as you can see from my earlier comments) but it felt more like "state" than "option", although it is kinda also "an option to use the cache" so can see it working either way.